workspace/src/github.com/darkcrux/go-ceph-rest-api/handler_pool.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+  "fmt"
+  "net/http"
+  "github.com/gorilla/mux"
+  "encoding/json"
+)
```

Comment:
a good convention is too group the packages by `built-in`, `built-in/extension`, and `3rd party`:

```
import (
    "fmt"

    "net/http"

    "encoding/json"
    "github.com/gorilla/mux"
)
```